docs/ci-integration/guides/gitlab-integration.md

```diff
@@ -0,0 +1,29 @@
+
+# GitLab CI/CD integration
+
+This example uses [GitLab CI/CD](https://docs.gitlab.com/ee/ci/) to build the Earthly target `+build`.
+
+
+```yml
+# .gitlab-ci.yml
+
+image: docker
+services:
+  - docker:dind
+
+before_script:
+    - apk update && apk add git
+    - wget https://github.com/earthly/earthly/releases/download/v0.6.1/earthly-linux-amd64 -O /usr/local/bin/earthly

```

Comment:
Seems that this is one version behind - mind updating to `v0.6.2`? Can you also check `release/README.md` to see if the process to update versions in docs would cover this page too?